packages/core/src/components/ChoiceList/Choice.jsx

```diff
@@ -79,9 +79,9 @@ Choice.propTypes = {
    */
   inversed: PropTypes.bool,
   /**
-   * Displays the label on the left side of the check
+   * Displays the checkbox on the right side of the label
    */
-  labelOnLeft: PropTypes.bool,
+  right: PropTypes.bool,
```

Comment:
I'm torn on this name. `labelOnLeft` makes it more obvious to me that it expects a boolean value and makes clear what behavior is expected when the prop is present. (I understand why this change was made though).

A few other ideas:
- I'm wondering if we should instead support the option to pass in additional class names for the `input` (e.g. `inputClassName`), which isn't currently supported but might be useful, especially if/when we add another modifier for making the checkbox small. (Note: The Button component includes a `size` prop)
- Another idea: Having a prop like `inputPlacement` which accepts a value of either `left` (default) or `right`.

I think I'm leaning more for `inputClassName` since that seems like nice functionality to have either way, but not a strong preference.

Reply:
I think we should have the `inputClassName` anyway, but I would like to also have a prop specifically for the label placement, because half the point of having these react classes is that we abstract away some of these long class names and give people a nicer interface for defining these things. I think `inputPlacement` as a string is easier than coming up with a good name for a boolean.